documentation/api.md

```diff
@@ -652,6 +652,51 @@ message | String | (Required) Comment contents as raw Markdown text. New lines r
 #### Response
 No `data` is returned.
 
+##View a Backup's Comments
+>><h4> Example Response </h4>
+> ```
+python
+import requests
+url = "https://okpy.org/api/v3/backups/a24x23/comment?access_token={}"
+access_token = 'test'
+r = requests.get(url.format(access_token))
+response = r.json()
+```
+>><h4> Response </h4>
+```
+{
+  "data": {
+    "comments":[
+      {
+        'id':
```

Comment:
It's fine if you make something up for the `id`. Maybe `'wdLGDb',` (it should just be different from the backup_id so people don't confuse this field for the backup id)